development.ini

```diff
@@ -68,7 +68,7 @@ db_schema = public
 # For virtuoso:
 # db_schema = assembl
 db_database = assembl
-# db_host = localhost
+db_host = database
```

Comment:
It looks that this change would break the current installation procedure of Assembl. It would worry me if Assembl did not allow a docker-less installation anymore, and forced the use of docker (and other tools of its ecosystem) instead. Could we find a way to support both? For example, at some point @maparent was working on the development of a .ini overriding/cascading mechanism. Maybe this could help? (a specific docker ini file would override development.ini)